pdcupdater/tests/handler_tests/test_depchain_rpms.py

```diff
@@ -0,0 +1,253 @@
+import mock
+
+import pdcupdater.utils
+from pdcupdater.tests.handler_tests import (
+    BaseHandlerTest, mock_pdc
+)
+
+
+class TestBuildtimeDepIngestion(BaseHandlerTest):
+    maxDiff = None
+    handler_path = 'pdcupdater.handlers.depchain.rpms:NewRPMBuildTimeDepChainHandler'
+    config = {}
+
+    def test_can_handle_buildsys_tag_message(self):
+        idx = '2016-5af8c0c2-9acd-4cf4-afa6-c07649bb5561'
+        msg = pdcupdater.utils.get_fedmsg(idx)
+        result = self.handler.can_handle(msg)
+        self.assertEquals(result, True)
```

Comment:
Not a big deal but you can use this instead:
self.assertTrue(result)

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertTrue